lib/rspec/core/configuration.rb

```diff
@@ -781,23 +783,62 @@ def full_backtrace=(true_or_false)
       # Returns the configuration option for color, but should not
       # be used to check if color is supported.
       #
+      # @deprecated No longer recommended because of complex behavior. Instead,
+      #   rely on the fact that TTYs will display color by default, or use
+      #   {#force_color} to display color on a non-TTY output.
       # @see color_enabled?
       # @return [Boolean]
       def color
         value_for(:color) { @color }
       end
 
+      # Returns the configuration option for forcing color, but should not
+      # be used to check if color is supported.
```

Comment:
Do we want `force_color` and `no_color` to have `@see` annotations pointing to `color`? Since `color` is deprecated, maybe we don't need to draw attention to it.